source/includes/steps-import.yaml

```diff
@@ -19,25 +21,57 @@ content: |
 
   .. figure:: /images/compass/import-data-dialog.png
      :alt: Import data dialog
+     :width: 60%
 
 ---
-title: Choose the appropriate file type of the source data file. 
+title: Select the location of the source data file under 
+       :guilabel:`Select File`.
+level: 4
+ref: click-file-location
+content: ""
+
+---
+title: Choose the appropriate file type.
 level: 4
 ref: choose-file-type
 content: |
-  Under :guilabel:`Select Input File Type`, select either JSON or CSV. 
+
+  Under :guilabel:`Select Input File Type`, select either 
+  :guilabel:`JSON` or :guilabel:`CSV`. 
+
+  If you are importing a CSV file, you may specify fields to import and 
+  the types of those fields under :guilabel:`Specify Fields and Types`:
+
+  .. figure:: /images/compass/import-csv-options.png
+     :alt: Select fields to import from a CSV file
+     :width: 60%
+
+  To exclude a field from a CSV file you are importing, uncheck the 
+  checkbox next to that field name. To select a type for a field, use 
+  the dropdown menu below that field name.
 
 ---
-title: Select the location of the source data file under :guilabel:`Select File`.
+title: Configure import options.
 level: 4
-ref: click-file-location
+ref: import-options
 content: |
-  .. raw:: html
+  Under :guilabel:`Options`, select the import options for your use 
+  case. 
+  
+  If you are importing a CSV file, you may select how your data 
+  is delimited.
+
+  If checked, :guilabel:`Ignore empty strings` drops document fields 
```

Comment:
It's not immediately clear that the next two settings apply to both JSON and CSV imports. Maybe tack on a sentence to the end of both like "You can set this setting for both JSON and CSV imports." or something like that.